junitparser/cli.py

```diff
@@ -56,6 +57,11 @@ def _parser(prog_name=None):  # pragma: no cover
     merge_parser.add_argument(
         "output", help='Merged XML Path, setting to "-" will output console'
     )
+    merge_parser.add_argument(
+        "--suiteName",
+        help="Name added to <testsuites>, default Test suites.",
+        default="Test suites",
```

Comment:
Better to leave `default` as an empty string.

Reply:
Good point - my thinking was just that it was probably good practice to name the suites, but that's probably just my opinion (and as you say, it could be a breaking change)